src/components/storyCard/storyCard.module.scss

```diff
@@ -0,0 +1,109 @@
+.container {
+    max-width: 450px;
+    padding: 15px;
+    display: flex;
+    align-items: stretch;
+    align-content: center;
+    justify-content: space-between;
+    border: 5px solid #E0E0E0;
+    border-radius: 25px;
+}
+
+.title {
+    margin: 20px;
+    color: #041287;
+    font-size: 25px;
+}
+
+.description {
+    margin: 0px 20px 20px 20px;
+    color: #AEAEAE;
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
+
+.taskInfo {
+    font-size: 16px;
+    display: flex;
+    flex-direction: column;
+}
+
+.taskDue {
+    display: flex;
+    align-items: center;
+}
+
+.taskDue > img {
+    margin-right: 10px;
+}
+
+.dateInfo {
+    margin: 20px;
+}
+
+.taskStarted {
+    color: #AEAEAE;
+    margin-bottom: 8px;
+}
+
+.subtasks {
+    margin: 20px;
+}
+
+.subtask {
+    background-color: #F9F9F9;
+    margin: 8px 8px 8px 0px;
+    padding: 4px 6px 4px 10px;
+    border-radius: 7px;
+    border: 2px solid #E0E0E0;
+    color: #041287;
+    display: flex;
+    justify-content: space-between;
+}
+
+.subtaskStatus {
+    margin-left: 8px;
+    color: #01A338;
+}
+
+.taskStatus {
+    margin: 10px;
+    font-size: 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.statusInfo {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.ownerInfo {
+    display: flex;
+    align-items: center;
+}
+
+.ownerImage {
+    border-radius: 50%;
+    max-height: 50px;
+    margin-left: 8px;
+}
+
+.toBeDecided {
+    background-color: #F9F9F9;
+    font-size: 15px;
+    border-radius: 7px;
+    border: 2px solid #E0E0E0;
+    color: #041287;
+    justify-content: space-between;
+}
+
+.toBeDecided > p {
+    margin: 5px;
+}
+
+.dueDate {
+    margin-left: 10px;
+}
```

Comment:
One question, is your tab size set to 4? or is it 2?
I think everywhere we are using the tab size of 2 .. so if it's 4, can you please change it to 2 and re-format this?